reconcile/openshift_prometheusrules.py

```diff
@@ -0,0 +1,273 @@
+import json
+import semver
+import logging
+import traceback
+
+import utils.gql as gql
+import reconcile.openshift_base as ob
+import utils.jsonnet as jsonnet
+import utils.template as template
+
+from utils.defer import defer
+from utils.openshift_resource import (OpenshiftResource as OR)
+
+SLO_RULES = 'slo-rules.jsonnet.j2'
+
+PERFORMANCE_PARAMETERS_QUERY = """
```

Comment:
can we move this into queries.py?
it will allow us to re-use the query for the app-interface reporter for example.

Reply:
will do!

Reply:
fixed in e41c427